C++/C++.sublime-syntax

```diff
@@ -275,19 +275,19 @@ contexts:
       push: global-modifier
     # Take care of comments just before a function definition.
     - match: /\*
-      scope: punctuation.definition.comment.c
+      scope: punctuation.definition.comment.c++
```

Comment:
I'd vote to keep PRs small and focused on certain aspects. It is probably not always easy but still preferable.

This PR is about detecting "global constants".